### PR TITLE
Ethan - pokedex complete

### PR DESCRIPTION
**Pull request by VanguardAWS**

### CODE REVIEW

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 4: Vue Todos](https://github.com/PdxCodeGuild/class_swordfish/blob/main/4%20JavaScript/labs/Lab%2004%20Vue%20Todos.md)
**Completed**

[Lab 5: FavQs in Vue](https://github.com/PdxCodeGuild/class_swordfish/blob/main/4%20JavaScript/labs/Lab%2005%20Quotes%20in%20Vue.md)
**Needs Work**
* Need to clear out search boxes after each search so the previous 'tag' search doesn't show when the user is currently doing an 'author' search. See image below.
* Pagination needs to work. A user needs to be able to see the next and previous pages of quotes if they are available (>25 quotes).
* Need to provide an explicit key for the quote components. See image below.


image
image

**Review by brucestull — COMMENTED** [review]
[Lab 6: Mini-Capstone: Any API in Vue](https://github.com/PdxCodeGuild/class_swordfish/blob/main/4%20JavaScript/labs/Lab%2006%20Mini-Capstone.md)

NOTE: Waiting on student to present application. Will mark lab completed after student presents.

**Review by Faebaeee — CHANGES_REQUESTED** [review]
[Lab 04 - Django REST Framework](https://github.com/PdxCodeGuild/class_swordfish/blob/main/3%20Django/labs/04%20DRF%20Vue.md)
Try giving this lab a fix, Let me know if you have questions!
Screen Shot 2022-11-29 at 12 14 01 PM

**Review by brucestull — APPROVED** [review]
[Lab 5: FavQs in Vue](https://github.com/PdxCodeGuild/class_swordfish/blob/main/4%20JavaScript/labs/Lab%2005%20Quotes%20in%20Vue.md)
**Completed**

File: code/ethan/django_lab04/students_app/urls.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from .import views
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+router = DefaultRouter()
+router.register('students', views.StudentViewSet, basename='students')
+
+app_name = 'students_app'
+
+urlpatterns = [
+    path('/', views.index, name='index'),
```
Comment (brucestull) [inline code/ethan/django_lab04/students_app/urls.py:10]: image


This `/` needs to be removed and a slash `/` added to the path in `students.urls.py`:
* `path('/', views.index, name='index')` should be `path('', views.index, name='index')`
* `path('api', include('students_app.urls'))` should be `path('api/', include('students_app.urls'))`

## Current config results in the following:
image
image

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 04 - Django REST Framework](https://github.com/PdxCodeGuild/class_swordfish/blob/main/3%20Django/labs/04%20DRF%20Vue.md)
**Needs Work**
# Comments:
* See inline comments.

**Comment by brucestull** [comment]
[Lab 04 - Django REST Framework - Students API](https://github.com/PdxCodeGuild/class_swordfish/blob/main/3%20Django/labs/04%20Students%20API.md)
**Completed**

**Comment by brucestull** [comment]
[Lab 05 - Pokedex with Django REST Framework and Vue](https://github.com/PdxCodeGuild/class_swordfish/blob/main/3%20Django/labs/05%20Pokedex.md)
* `add` functions properly.

**Review by brucestull — APPROVED** [review]
# Labs completed see links below for specific lab numbers and names.
https://github.com/PdxCodeGuild/class_swordfish/pull/196#issuecomment-1335875971
https://github.com/PdxCodeGuild/class_swordfish/pull/196#issuecomment-1335918995

**Pull request merged**